### PR TITLE
Fixed make setup

### PR DESCRIPTION
**Pull request by mjvkeulen**
## Description
Fixed the order of the make setup script to once again work for fresh installs.

- Added `postgres-reset`, if you just want to run a reset of Postgres data without filling the DB with testing data
- CI will no longer run `preset-db` during setup (but still as a part of the test suite)
- Fixes Z#1187

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -77,7 +78,8 @@ setup-no-input:
 	# Reinstall nodejs dependencies.
 	npm install --prefix ./src/vue
 
-	make preset-db-no-input
+	make postgres-reset
```
Comment (Larspolo) [inline Makefile:81]: This can be `make postgres-init` right? `postgres-clean` is already run in `clean`

Comment (mjvkeulen) [reply Makefile:81]: yes good catch, will update

**Comment by mjvkeulen** [comment]
> # [Codecov](https://codecov.io/gh/eJourn-al/eJournal/pull/997?src=pr&el=h1) Report
> > Merging [#997](https://codecov.io/gh/eJourn-al/eJournal/pull/997?src=pr&el=desc) into [develop](https://codecov.io/gh/eJourn-al/eJournal/commit/0e6046880b9893ac9f1ab110373948a7fef1306c?el=desc) will **decrease** coverage by `0.02%`.
> > The diff coverage is `n/a`.
> 
> [Impacted file tree graph](https://codecov.io/gh/eJourn-al/eJournal/pull/997?src=pr&el=tree)
> 
> ```diff
> @@             Coverage Diff             @@
> ##           develop     #997      +/-   ##
> ===========================================
> - Coverage    89.57%   89.54%   -0.03%     
> ===========================================
>   Files           46       46              
>   Lines         4353     4353              
>   Branches       781      781              
> ===========================================
> - Hits          3899     3898       -1     
> - Misses         280      281       +1     
>   Partials       174      174              
> ```
> 
> [Impacted Files](https://codecov.io/gh/eJourn-al/eJournal/pull/997?src=pr&el=tree)	Coverage Δ	
> [src/django/VLE/tasks/beats/notifications.py](https://codecov.io/gh/eJourn-al/eJournal/pull/997/diff?src=pr&el=tree#diff-c3JjL2RqYW5nby9WTEUvdGFza3MvYmVhdHMvbm90aWZpY2F0aW9ucy5weQ==)	`94.25% <0.00%> (-1.15%)`	
> [Continue to review full report at Codecov](https://codecov.io/gh/eJourn-al/eJournal/pull/997?src=pr&el=continue).
> 
> > **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> > `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> > Powered by [Codecov](https://codecov.io/gh/eJourn-al/eJournal/pull/997?src=pr&el=footer). Last update [0e60468...9f3e388](https://codecov.io/gh/eJourn-al/eJournal/pull/997?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

Why does codecov mention a drop in a file which is not touched?

**Comment by engelhamer** [comment]
> > # [Codecov](https://codecov.io/gh/eJourn-al/eJournal/pull/997?src=pr&el=h1) Report
> > > Merging [#997](https://codecov.io/gh/eJourn-al/eJournal/pull/997?src=pr&el=desc) into [develop](https://codecov.io/gh/eJourn-al/eJournal/commit/0e6046880b9893ac9f1ab110373948a7fef1306c?el=desc) will **decrease** coverage by `0.02%`.
> > > The diff coverage is `n/a`.
> > 
> > [Impacted file tree graph](https://codecov.io/gh/eJourn-al/eJournal/pull/997?src=pr&el=tree)
> > 
> > ```diff
> > @@             Coverage Diff             @@
> > ##           develop     #997      +/-   ##
> > ===========================================
> > - Coverage    89.57%   89.54%   -0.03%     
> > ===========================================
> >   Files           46       46              
> >   Lines         4353     4353              
> >   Branches       781      781              
> > ===========================================
> > - Hits          3899     3898       -1     
> > - Misses         280      281       +1     
> >   Partials       174      174              
> > ```
> > 
> > [Impacted Files](https://codecov.io/gh/eJourn-al/eJournal/pull/997?src=pr&el=tree)	Coverage Δ	
> > [src/django/VLE/tasks/beats/notifications.py](https://codecov.io/gh/eJourn-al/eJournal/pull/997/diff?src=pr&el=tree#diff-c3JjL2RqYW5nby9WTEUvdGFza3MvYmVhdHMvbm90aWZpY2F0aW9ucy5weQ==)	`94.25% <0.00%> (-1.15%)`	
> > [Continue to review full report at Codecov](https://codecov.io/gh/eJourn-al/eJournal/pull/997?src=pr&el=continue).
> > 
> > > **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> > > `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> > > Powered by [Codecov](https://codecov.io/gh/eJourn-al/eJournal/pull/997?src=pr&el=footer). Last update [0e60468...9f3e388](https://codecov.io/gh/eJourn-al/eJournal/pull/997?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).
> 
> Why does codecov mention a drop in a file which is not touched?

Weekly digest emails are only sent on mondays. Since it is not monday today, 1 specific line in the notification beats file was not triggered.

Funny but probably not desirable 😋

**Review by Larspolo — APPROVED** [review]

**Review by engelhamer — APPROVED** [review]
Awesome

**Pull request merged**